integration-tests/slack/notify-circleci-test-results.js

```diff
@@ -0,0 +1,77 @@
+const _ = require('lodash/fp')
+const { getFailedTestNamesFromArtifacts } = require('./circleci-utils')
+const { getMessageBlockTemplate } = require('./message-templates')
+const { postMessage } = require('./post-message')
+const testsInfo = require('./slack-notify-channels.json')
+
+
+/**
+ * Get array of Slack channel IDs for succeeded job notification
+ * @returns {Array[string]}
+ */
+const getAllSlackChannelsForPassedJob = () => {
+  return _.map(_.get('id'), testsInfo.pass)
```

Comment:
Nit: Again, I think this shorthand will work.
```suggestion
  return _.map('id', testsInfo.pass)
```

Reply:
Done.